### PR TITLE
[GE] Gizmo!

### PR DESCRIPTION
**Pull request by msDestiny14**
image

### CODE REVIEW

File: guiEditor/src/diagram/guiGizmo.tsx [hunk 1]
```diff
@@ -0,0 +1,532 @@
...
+        projected.x = round(projected.x);
+        projected.y = round(projected.y);
+        return projected;
+    }
+
+    // private _canvasSpaceToRTTSpace(node: Control, projected: Vector3) {
+    //     const camera = this.props.globalState.workbench._camera;
+    //     const scene = this.props.globalState.workbench._scene;
+    //     const engine = scene.getEngine();
+    //     const viewport = camera.viewport.toGlobal(engine.getRenderWidth(), engine.getRenderHeight());
```
Comment (msDestiny14) [inline guiEditor/src/diagram/guiGizmo.tsx:259]: Can this be deleted?

Comment (RaananW) [reply guiEditor/src/diagram/guiGizmo.tsx:259]: yes, it can

**Review by RaananW — APPROVED** [review]
Make sure to resolve the conflicts.

**Comment by RaananW** [comment]
There is a doc issue

**Pull request merged**